consensus-types/blocks/execution.go

```diff
@@ -140,11 +140,21 @@ func (e executionPayload) Transactions() ([][]byte, error) {
 	return e.p.Transactions, nil
 }
 
+// TransactionsRoot --
```

Comment:
There are no tests for this file. I have some in [`develop...capella-state-processing`#diff-a4ffcfb9e9](https://github.com/prysmaticlabs/prysm/compare/develop...capella-state-processing#diff-a4ffcfb9e9d5ea43d14f3217b54c9d2502a8b33d05934d07628b07ff79394887)